Plugin/Model/ResourceModel/Quote/ExtensionAttributesPersistencePlugin.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ * @author      sean.templeton
+ */
+
+namespace ClassyLlama\AvaTax\Plugin\Model\ResourceModel\Quote;
+
+use Magento\Framework\Api\ExtensionAttribute\Config\Converter;
+use Magento\Framework\Api\ExtensionAttribute\JoinProcessorHelper;
+use Magento\Framework\Api\ExtensionAttributesFactory;
+use Magento\Framework\DataObject;
+use Magento\Framework\Model\AbstractModel;
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+
+class ExtensionAttributesPersistencePlugin
+{
+    /**
+     * @var JoinProcessorHelper
+     */
+    protected $joinProcessorHelper;
+
+    /**
+     * @var ExtensionAttributesFactory
+     */
+    protected $extensionAttributesFactory;
+
+    /**
+     * @param JoinProcessorHelper        $joinProcessorHelper
+     * @param ExtensionAttributesFactory $extensionAttributesFactory
+     */
+    public function __construct(
+        JoinProcessorHelper $joinProcessorHelper,
+        ExtensionAttributesFactory $extensionAttributesFactory
+    )
+    {
+        $this->joinProcessorHelper = $joinProcessorHelper;
+        $this->extensionAttributesFactory = $extensionAttributesFactory;
+    }
+
+    protected function getJoinDirectivesForType($extensibleEntityClass)
```

Comment:
Add DocBlock, including a comment(s) describing what this does

Reply:
Disregard this comment. I'm going to add all comments to the PR, not the individual commits.